### PR TITLE
[Astro] Update README

### PR DESCRIPTION
**Pull request by baderbuddy**
**Why This PR?**
_a short description of why this PR is needed_
Updates the README to use helm-docs

Fixes #

**Changes**
Changes proposed in this pull request:

* Adds documentation to Values.yaml and moves to using helm-docs

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Review by sudermanjr — APPROVED** [review]
Approving my little portion

File: stable/astro/values.yaml [hunk 1]
```diff
@@ -1,35 +1,56 @@
...
   serviceAccount:
+    # -- If true, a service account will be created. If false, you must set `deployment.serviceAccount.name`.
     create: true
-    # name: ExistingServiceAccountName
+    # -- The name of an existing service account to use.
+    name:
+  # -- The number of replicas to use.
+  replicas: 2
 secret:
+  # -- If true, a secret with API credentialss will be created. If false, you must set `secret.name`
```
Comment (lucasreed) [inline stable/astro/values.yaml]:
```suggestion
  # -- If true, a secret with API credentials will be created. If false, you must set `secret.name`
```

File: stable/astro/values.yaml [hunk 2]
```diff
@@ -1,35 +1,56 @@
...
   create: true
-  # name: ExistingSecretName
+  # -- The name of an existing secret to mount to the container.
+  name:
```
Comment (lucasreed) [inline stable/astro/values.yaml:42]: Is it OK to leave this blank? previously it was commented out.

File: stable/astro/values.yaml [hunk 3]
```diff
@@ -1,35 +1,56 @@
...
+  # -- If true, RBAC resources will be created.
   create: true
 deployment:
-  replicas: 2
   serviceAccount:
+    # -- If true, a service account will be created. If false, you must set `deployment.serviceAccount.name`.
     create: true
-    # name: ExistingServiceAccountName
+    # -- The name of an existing service account to use.
+    name:
```
Comment (lucasreed) [inline stable/astro/values.yaml:35]: Same question here about it being blank and previously being commented out. Is this ok?

**Review by lucasreed — COMMENTED** [review]
One small spelling fix and a question about a particular value with no default that is not commented

File: stable/astro/values.yaml (hunk 2 above)
Comment (baderbuddy) [reply stable/astro/values.yaml:42]: This is a weird syntax to me. But apparently that's one of the ways to represent nulls in Yaml. This way it still won't have a value.  This way we can still document it.

**Review by lucasreed — APPROVED** [review]

**Pull request merged**